services/replicator/metadataReconciler.go

```diff
@@ -803,6 +804,12 @@ func (r *metadataReconciler) reconcileDestExtent(destUUID string, localExtents m
 					OriginZone:      common.StringPtr(remoteZone),
 				},
 			}
+
+			// if the remote extent has cg visibility set (a merged dlq extent), propagate that field too.
+			if remoteExtentStats.IsSetConsumerGroupVisibility() {
```

Comment:
what happens, when you have a DC_LOCAL consumer group for a multi-zone destination ? In that case, won't this code try to replicate the dlq extents belonging to the DC_LOCAL consumer group ?